palo_alto_pan_os/komand_palo_alto_pan_os/actions/set_address_object/action.py

```diff
@@ -13,12 +14,52 @@ def __init__(self):
                 input=SetAddressObjectInput(),
                 output=SetAddressObjectOutput())
 
+    def determine_address_type(self, address):
+        if re.search('[a-zA-Z]', address):
+            return "fqdn"
+        if re.search('/', address):
+            return "ip-netmask"
+        if re.search('-', address):
+            return "ip-range"
+        return "ip-netmask"
+
+    def match_whitelist(self, address, whitelist, object_type):
+        if object_type == "fqdn":
+            if address in whitelist:
+                self.logger.info(f" Whitelist matched\nIP {address} was found in whitelist")
```

Comment:
Minor nit, reads a little weird when it's comparing a /32 to an IP but it also makes sense.

```
IP 1.1.1.2 was found in 1.1.1.2/32
```

Reply:
Changed it...on that line, it will be an fqdn, not an IP.